### ISSUE TITLE
Follow the ARIA design pattern for the autocomplete component

### ISSUE THREAD

**Issue by Veyfeyken** [digipolisantwerp/acpaas-ui_react#199]
## Problem

The autocomplete component lacks the necessary ARIA roles and attributes to be accessible with assistive technology such as screenreaders.

## Suggestion

Follow the ARIA design pattern for Combobox: List autocomplete with automatic selection.

## Articles and demo

* [ARIA design pattern: combobox](https://www.w3.org/TR/wai-aria-practices-1.2/#combobox)
* Sarah Higley:
    * [Select your poison part 1](https://www.24a11y.com/2019/select-your-poison/)
    * [Select your poison part 2](https://www.24a11y.com/2019/select-your-poison-part-2/)
